omod/src/main/webapp/main.jsp

```diff
@@ -77,9 +77,13 @@
 	</table>
 	<div id="tabs">
 		<ul>
-			<li><a
-				href="opdEntry.htm?patientId=${patient.patientId }&opdId=${opd.conceptId }&queueId=${queueId}"
-				title="OPD entry"><span> Clinical Notes</span> </a></li>
+			<c:set var="compare_status" value="Not Paid Reg Fee"/>
+			<c:if test="${(visitStatus != compare_status)}">
```

Comment:
@franqq did you change your mind about this, thought you decided to check for the presence of `visitStatus` and not the value?